docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -4,6 +4,160 @@ sidebar_label: With Argo CD
 
 # Advanced Installation with Argo CD
 
+This section outlines the various advanced installation options available for deploying Kargo on a Kubernetes cluster using Argo CD.
+
+:::warning
+This section assumes that you have already installed any dependencies or prerequisites required for running Kargo on a Kubernetes cluster. Please see the [installation guide](../../40-operator-guide/10-basic-installation.md#prerequisites) for more information.
+:::
+
+## Argo CD Helm Application
+
+The most common way to deploy Kargo using Argo CD is to create a Helm `Application`, and use the `.spec.source.helm.parameters` section to specify any parameters you may need. This is the most straightforward way to deploy Kargo using Argo CD.
```

Comment:
The notion of a Helm Application and especially a Helm `Application` seems like it might be slightly confusing terminology.

I would probably say something about Kargo being packaged as a Helm chart and that you'll show how to use an Argo CD `Application` to deploy Kargo straight from its own chart repository with two different options for how non-default configuration values can be specified.

Reply:
ACK, that's a good point; because in either case you're still using the Helm chart